dsd/vienna_nupack.py

```diff
@@ -429,7 +429,7 @@ def domain_equal_strength(seq: str, temperature: float, sodium: float,
 
 
 def domain_no_sec_struct(seq: str, temperature: float, sodium: float,
```

Comment:
I actually forgot this function was here. It's not used in any examples. I should probably just get rid of it. It's not how we'd use a constraint in the library now anyway. If we were using it as a `DomainConstraint` in the library, we'd want to return a float, for example the value

`max(0.0, seq_sec_struc - individual) + max(0.0, seq_wc_sec_struc - individual)`

But it should be structured more like https://github.com/UC-Davis-molecular-computing/dsd/blob/716b247e6b224872a35eb88553b31bbf0fa38f65/dsd/constraints.py#L3201

and be a function that returns a `DomainConstraint` object.

Can we instead just delete this function instead (and the one above it, `domain_equal_strength`)?

Reply:
I actually removed them in the dev branch. If you wanted something like this functionality, we can write a `DomainConstraint` (or `SequenceConstraint` to give to a `DomainPool` if you really wanted it to be a hard Boolean constraint.

Reply:
Makes sense! Thank you!